### ISSUE TITLE
Could oneget customize installation directories for softwares or libs？

### ISSUE THREAD

**Issue by Tim-Lei** [OneGet/oneget#127]
I noticed that chocolatey would install packages in a default dir. And it does not allow to change it. What I want to do is to plan well the destination for different packages at installation phase. For instance, I would like to install all portable softwares in a 'D:\PTB' folder and then I can easily move them to another machine or copy back after OS restore or re-install without configurating the settings again. 

So I think supporting to customize the intallation directories would be a really nice option for automatic package management.Does oneget support this or will do?

**Comment by fearthecowboy** [comment]
The current implementation of the chocolatey provider is a prototype, and will be removed fairly shortly. The official chocolatey provider for OneGet should be coming from the chocolatey community, and should support the same features that the chocolatey exe supports.

**Comment by Tim-Lei** [comment]
OK. I thought oneget rewrote chocolatey with C# and could support some customization. Should I post this issue to chocolatey community? :)

**Comment by fearthecowboy** [comment]
Well, I did write a clean-room Chocolatey implementation as a proof-of-concept; the Chocolatey provider in OneGet is currently that proof-of-concept. Rob Reynolds is the author of the Official Chocolatey, and he is building a new version written in c#. 

It's not in the best interest to have two completely independent implementations of Chocolatey, since one would always be playing catch-up to the other. My prototype implementation is lacking a lot of the features that Chocolatey has, and rather than spend a bunch of wasted time catching up, we're deferring to the Official Chocolatey project.

In order for OneGet to use that version of Chocolatey, a connector needs to be created (what we call a 'package provider'). The effort to create the provider for the Official Chocolatey for OneGet is underway, but not yet complete.

Once the Official Chocolatey provider is complete, OneGet should be able to include it in the list of package providers, so that users can use it.  

It won't do any good to post that back to the Chocolatey project, since you're not actually using their code yet. 

Bottom line: you're in kindof a limbo right now with Chocolatey, where the current implementation isn't going to get fixed, and the new one's not ready yet.
